component/vdr/orb/vdr.go

```diff
@@ -423,16 +442,24 @@ func (v *VDR) Update(didDoc *docdid.Doc, opts ...vdrapi.DIDMethodOption) error {
 
 	// check recover option
 	if didMethodOpts.Values[RecoverOpt] != nil {
-		if didMethodOpts.Values[AnchorOriginOpt] == nil {
-			return fmt.Errorf("anchorOrigin opt is empty")
+		// Select domain
+		domain, errChoose := v.selectDomainSvc.Choose(v.domains)
```

Comment:
We should probably not switch anchor origin domain during recovery unless this anchor origin change is specified by the client. If anchor origin is not provided by the client I would use the current anchor origin for that document @fqutishat @bstasyszyn

Reply:
@sandrask make sense. I change it